app/services/notification_generator.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Multiple children for expiring notification? One for each kid or a single one?
```

Comment:
Note: we have the answer to this now - we want one notification per child, let's remove this comment.